sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -3294,22 +3294,22 @@
           "values": [
             {
               "label": "Student went from full-time to part-time studies (vice-versa) or started a different program",
-              "value": "changeInIntensity",
+              "value": "Change in intensity",
```

Comment:
Can these values load dynamically from Vue by using the enum so that we can avoid duplication of the same enum here?

Reply:
i belive these are formio key (api value)

Reply:
My point was to check if it is possible to load these radio values dynamically from vue.
![image](https://user-images.githubusercontent.com/54600590/225772874-ba6cb845-ee49-4fb5-8c1c-d0adee582c48.png)

Reply:
We can, but I am not sure if we should. If they were used as text and value I would say 100% we should, in the current scenario, I am not sure.